ev3/legoev3_motor.c

```diff
@@ -57,12 +57,11 @@
 #include "../motors/ev3_motor.h"
 
 #define TACHO_MOTOR_POLL_NS	2000000	/* 2 msec */
+#define TACHO_MOTOR_POLL_MS	2000	/* 2 msec */
```

Comment:
Actually, I just ended up setting `TACHO_MOTOR_POLL_MS` to 2 and then used the kernel conversion factors for NSEC_PER_MSEC and USEC_PER_MSEC